tools/pipelines-tasks/AVDAppAttachPublish/ReportExceptionTelemetry.ps1

```diff
@@ -0,0 +1,43 @@
+<#
```

Comment:
File name nomenclature, there is a mix and match for the PS script names. Some are lower case, and some are upper case, is there a convention you are following?

Reply:
Using pascal case convention overall. changed publishAVD.ps1 as well to PublishAVD.ps1